src/common/responses/example-responses/PostOrgsSuccessResponse.yml

```diff
@@ -0,0 +1,19 @@
+  successResponse:
```

Comment:
Why store these in separate files? Do you expect to reuse them elsewhere?

Reply:
Some of the other responses are extremely large and I wanted to separate those into different files to keep the file cleaner.